src/templates/globals/header/index.tsx

```diff
@@ -2,7 +2,7 @@ import { TopNav } from './TopNav'
 
 export const Header = () => {
   return (
-    <header className="header" role="banner">
+    <header className="header merger" role="banner">
```

Comment:
this fixes the header styles, but will you move this onto the `<body>` tag in `_document.tsx` instead? That's where I'm seeing it on va.gov, other non-header components may also need that class in their scope

Reply:
good catch, fixed.